### PR TITLE
BROKERS: Change to SelectAllAsync

### LINKED ISSUE #135: BROKERS: Change SelectAllSources to SelectAllSourcesAsync
https://www.youtube.com/watch?v=wCB0h3Smv-Q

```cs
        public async ValueTask<IQueryable<Source>> SelectAllSourcesAsync() => 
            await ValueTask.FromResult(ReadAll<Source>());
```

### LINKED ISSUE #136: BROKERS: Change SelectAllRepositories to SelectAllRepositoriesAsync
https://www.youtube.com/watch?v=wCB0h3Smv-Q

```cs
        public async ValueTask<IQueryable<Repository>> SelectAllRepositoriesAsync() => 
            await ValueTask.FromResult(ReadAll<Repository>());
```

### LINKED ISSUE #137: BROKERS: Change SelectAllContributors to SelectAllContributorsAsync
https://www.youtube.com/watch?v=wCB0h3Smv-Q

```cs
        public async ValueTask<IQueryable<Contributor>> SelectAlContributorsAsync() => 
            await ValueTask.FromResult(ReadAll<Contributor>());
```

### LINKED ISSUE #138: BROKERS: Change SelectAllContributorTypes to SelectAllContributorTypesAsync
https://www.youtube.com/watch?v=wCB0h3Smv-Q

```cs
        public async ValueTask<IQueryable<ContributorType>> SelectAllContributorTypesAsync() => 
            await ValueTask.FromResult(ReadAll<ContributorType>());
```

### LINKED ISSUE #139: BROKERS: Change SelectAllContributions to SelectAllContributionsAsync
https://www.youtube.com/watch?v=wCB0h3Smv-Q

```cs
        public async ValueTask<IQueryable<Contribution>> SelectAllContributionsAsync() => 
            await ValueTask.FromResult(ReadAll<Contribution>());
```

### PR DESCRIPTION
**Pull request by Lezune**
Closes #135 
Closes #136 
Closes #137 
Closes #138 
Closes #139

### CODE REVIEW

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.cs [hunk 1]
```diff
@@ -55,8 +55,7 @@ private async ValueTask<T> InsertAsync<T>(T @object)
 
             return @object;
         }
-
-        private async ValueTask<IQueryable<T>> SelectAllAsync<T>() where T : class => 
+        private IQueryable<T> SelectAll<T>() where T : class =>
             this.Set<T>();
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Storages/StorageBroker.cs:59]:
```suggestion
        private async ValueTask<IQueryable<T>> SelectAllAsync<T>() where T : class => 
            await ValueTask.FromResult(this.Set<T>());
```

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Sources.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ internal partial class StorageBroker
             InsertAsync(source);
 
         public async ValueTask<IQueryable<Source>> SelectAllSourcesAsync() => 
-            await SelectAllAsync<Source>();
+            await ValueTask.FromResult(SelectAll<Source>());
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Storages/StorageBroker.Sources.cs:22]:
```suggestion
            await SelectAllAsync<Source>();
```

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Repositories.cs [hunk 3]
```diff
@@ -18,7 +18,7 @@ internal partial class StorageBroker
             await InsertAsync(repository);
 
         public async ValueTask<IQueryable<Repository>> SelectAllRepositoriesAsync() =>
-            await SelectAllAsync<Repository>();
+            await ValueTask.FromResult(SelectAll<Repository>());
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Storages/StorageBroker.Repositories.cs:21]:
```suggestion
            await SelectAllAsync<Repository>();
```

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributors.cs [hunk 4]
```diff
@@ -18,7 +18,7 @@ internal partial class StorageBroker
             InsertAsync(contributor); 
 
         public async ValueTask<IQueryable<Contributor>> SelectAllContributorsAsync() => 
-            await SelectAllAsync<Contributor>(); 
+            await ValueTask.FromResult(SelectAll<Contributor>());
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributors.cs:21]:
```suggestion
            await SelectAllAsync<Contributor>();
```

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.cs [hunk 5]
```diff
@@ -18,7 +18,7 @@ internal partial class StorageBroker
             await InsertAsync(contribution);
 
         public async ValueTask<IQueryable<Contribution>> SelectAllContributionsAsync() =>
-            await SelectAllAsync<Contribution>();
+            await ValueTask.FromResult(SelectAll<Contribution>());
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.cs:21]:
```suggestion
            await SelectAllAsync<Contribution>();
```

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.ContributionTypes.cs [hunk 6]
```diff
@@ -18,7 +18,7 @@ internal partial class StorageBroker
             await InsertAsync(contributionType);
 
         public async ValueTask<IQueryable<ContributionType>> SelectAllContributionTypesAsync() =>
-            await SelectAllAsync<ContributionType>();
+            await ValueTask.FromResult(SelectAll<ContributionType>());
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Storages/StorageBroker.ContributionTypes.cs:21]:
```suggestion
            await SelectAllAsync<ContributionType>();
```

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.cs (hunk 5 above)
Comment (Lezune) [reply GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.cs]: This has been resolved.

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributors.cs (hunk 4 above)
Comment (Lezune) [reply GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributors.cs]: This has been resolved.

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Repositories.cs (hunk 3 above)
Comment (Lezune) [reply GitFyle.Core.Api/Brokers/Storages/StorageBroker.Repositories.cs]: This has been resolved.

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.Sources.cs (hunk 2 above)
Comment (Lezune) [reply GitFyle.Core.Api/Brokers/Storages/StorageBroker.Sources.cs]: This has been resolved.

File: GitFyle.Core.Api/Brokers/Storages/StorageBroker.cs (hunk 1 above)
Comment (Lezune) [reply GitFyle.Core.Api/Brokers/Storages/StorageBroker.cs]: This has been resolved.

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**